CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * Moved `RouteController.tunnelSimulationEnabled` to `TunnelIntersectionManager.tunnelSimulationEnabled`. ([#1489](https://github.com/mapbox/mapbox-navigation-ios/pull/1489))
 * Deprecated `NavigationViewController.usesNightStyleInsideTunnels`. Style switching is enabled as a side effect of `TunnelIntersectionManager.tunnelSimulationEnabled`, which is set to `true` by default. ([#1489](https://github.com/mapbox/mapbox-navigation-ios/pull/1489))
 * Added `RouteControllerDelegate.routeControllerWillDisableBatteryMonitoring(_:)` which allows developers control whether battery monitoring is disabled when `RouteController.deinit()` is called.
+* Setting `NavigationLocationManager` to a custom value now works again. [#1481](https://github.com/mapbox/mapbox-navigation-ios/pull/1481)
```

Comment:
Should be `NavigationLocationManager.desiredAccuracy`. Fixed in 4784fdaf1d04a5f616fbf3722a5bd73c1069803b.